Build fails for 6.0.2: `sys.c: error: 'env' undeclared (first use in this function); did you mean 'Genv'?`

`radare2` 6.0.2 fails to build on macOS with:
```
/opt/local/bin/gcc-mp-14 -c -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/include -pipe -I/opt/local/libexec/openssl3/include -Os -arch ppc -MD -fno-common   -fPIC -Wall -g -DR2__UNIX__=1 -DR2_PLUGIN_INCORE -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/..//shlr -DUSE_RLZ4 -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -fvisibility=hidden -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -DHAVE_FORK=1 -DUSE_R2=1 -I../../shlr -o sys.o sys.c
/opt/local/bin/gcc-mp-14 -c -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/include -pipe -I/opt/local/libexec/openssl3/include -Os -arch ppc -MD -fno-common   -fPIC -Wall -g -DR2__UNIX__=1 -DR2_PLUGIN_INCORE -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/..//shlr -DUSE_RLZ4 -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -fvisibility=hidden -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -DHAVE_FORK=1 -DUSE_R2=1 -I../../shlr -o buf.o buf.c
/opt/local/bin/gcc-mp-14 -c -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/include -pipe -I/opt/local/libexec/openssl3/include -Os -arch ppc -MD -fno-common   -fPIC -Wall -g -DR2__UNIX__=1 -DR2_PLUGIN_INCORE -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/..//shlr -DUSE_RLZ4 -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -fvisibility=hidden -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -I/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/../shlr/../subprojects/sdb/include -DHAVE_FORK=1 -DUSE_R2=1 -I../../shlr -o sys_w32.o sys_w32.c
sys.c: In function 'r_sys_clearenv':
sys.c:410:14: error: 'env' undeclared (first use in this function); did you mean 'Genv'?
  410 |         if (!env) {
      |              ^~~
      |              Genv
sys.c:410:14: note: each undeclared identifier is reported only once for each function it appears in
sys.c: In function 'r_sys_pid_to_path':
sys.c:1308:9: note: '#pragma message: TODO getpidproc'
 1308 | #pragma message("TODO getpidproc")
      |         ^~~~~~~
make[3]: *** [sys.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory `/opt/local/var/macports/build/radare2-b1567b87/work/radare2-6.0.2/libr/util'
make[2]: *** [prelib-build] Error 2
```

@trufae FYI

--- a/libr/util/sys.c
+++ b/libr/util/sys.c
@@ -407,13 +407,15 @@ R_API bool r_sys_clearenv(void) {
 #if R2__UNIX__
 #if __APPLE__ && !HAVE_ENVIRON
 	/* do nothing */
-	if (!env) {
+	if (!Genv) {
 		r_sys_env_init ();
 		return true;
 	}
-	char **e = env;
-	while (*e) {
-		*e++ = NULL;
+	char **e = Genv;
+	if (e) {
+		while (*e) {
+			*e++ = NULL;
+		}
 	}
 #else
 	if (!environ) {

